### PR TITLE
cppSlint and rustSlint production Dockerfile fixes + Github Actions Checks

### LINKED ISSUE #95: `rustSlint` production image does not build correctly
The `torizon/wayland-base` is set with a wrong base tag

### LINKED ISSUE #97: Create a task to run the build of the production Dockerfile without the need to push it to the registry
In some cases it's good to have this task to test the sanity of the production Dockerfile for example.

### LINKED ISSUE #101: cppQML: build-release-arm64: error failed to solve: torizon/wayland-base:3.0.8-bookworm: docker.io/torizon/wayland-base:3.0.8-bookworm: not found
After creating a project for Qt 6 C++ from template, and trying to run the task `build-release-arm64`, I get the following error:

```
Executing task: DOCKER_HOST= docker compose build --build-arg IMAGE_ARCH=arm64 qt-gui-optimization 

WARN[0000] The "GPU" variable is not set. Defaulting to a blank string. 
WARN[0000] The "LOCAL_REGISTRY" variable is not set. Defaulting to a blank string. 
[+] Building 1.9s (5/5) FINISHED                                                                                                                                                
 => [qt-gui-optimization internal] load .dockerignore                                                                                                                      0.0s
 => => transferring context: 56B                                                                                                                                           0.0s
 => [qt-gui-optimization internal] load build definition from Dockerfile                                                                                                   0.0s
 => => transferring dockerfile: 3.27kB                                                                                                                                     0.0s
 => CANCELED [qt-gui-optimization internal] load metadata for docker.io/torizon/wayland-base:3.0.8                                                                         1.8s
 => ERROR [qt-gui-optimization internal] load metadata for docker.io/torizon/wayland-base:3.0.8-bookworm                                                                   1.8s
 => [qt-gui-optimization auth] torizon/wayland-base:pull token for registry-1.docker.io                                                                                    0.0s
------
 > [qt-gui-optimization internal] load metadata for docker.io/torizon/wayland-base:3.0.8-bookworm:
------
failed to solve: torizon/wayland-base:3.0.8-bookworm: docker.io/torizon/wayland-base:3.0.8-bookworm: not found

 *  The terminal process "/usr/bin/bash '-c', 'DOCKER_HOST= docker compose build --build-arg IMAGE_ARCH=arm64 qt-gui-optimization'" terminated with exit code: 17. 
 *  Terminal will be reused by tasks, press any key to close it.
```

By inspecting the above logs, it seems that it uses the wrong tag `3.0.8-bookworm`, whereas it should be `3.0.8`.

By looking at the Dockerfile, maybe there are two reasons:

- https://github.com/toradex/vscode-torizon-templates/blob/bookworm/cppQML/Dockerfile#L6 --> the tag is wrong
- https://github.com/toradex/vscode-torizon-templates/blob/bookworm/cppQML/Dockerfile#L28 --> the base image is wrong

### PR DESCRIPTION
**Pull request by microhobby**
fixes #95 
fixes #101 
fixes #97

### CODE REVIEW

**Comment by microhobby** [comment]
I'm setting this as draft because I think that is time to take advantage to implement some Github Actions tests to not have this fails again.

File: rustSlint/Dockerfile [hunk 1]
```diff
@@ -107,6 +107,6 @@ ENV SLINT_BACKEND=winit-skia
 ENV SLINT_FULLSCREEN=1
 
 # ADD YOUR ARGUMENTS HERE
-CMD [ "./app" ]
+CMD [ "./__change__" ]
```
Comment (tronical) [inline rustSlint/Dockerfile]: I guess the environment variable in like 74 is unused then :)

**Review by tronical — COMMENTED** [review]
Thank you for the fixes! We’re planning a release next Monday so the timing of this is excellent;)

File: rustSlint/Dockerfile (hunk 1 above)
Comment (microhobby) [reply rustSlint/Dockerfile]: oh yeah, you are right, I'm rollback this change. Thanks for the review!

**Comment by microhobby** [comment]
Hey @tronical I bumped the Slint version for the Dockerfiles in the 0af18454b33d74c290e460eec259f3f455e0d176

Let me know if I need to bump it into some other template file.

**Review by tronical — APPROVED** [review]

**Pull request merged**